detox/test/e2e/utils/rnSkipper.js

```diff
@@ -0,0 +1,21 @@
+const rn = require('../../../src/utils/rn-consts/rn-consts').rnVersion.minor;
+
+/** @type {import('jest-environment-emit').EnvironmentListenerFn} */
+const listener = ({ testEvents }) => {
+  testEvents
+    .on('start_describe_definition', ({ event: { blockName }, state: { currentDescribeBlock }}) => {
+      const match = blockName.match(/@rn(\d+)/i);
+      if (match && match[1] != rn) {
+        currentDescribeBlock.mode = 'skip';
+      }
```

Comment:
So that means we can use multiple `@rn` versions on a test, right?

Reply:
Actually, no, it is 1 version only at the moment.